app/assets/javascripts/client/views/assessments/permissions/manage_permission.html

```diff
@@ -60,7 +60,7 @@
             <p class="info">
               <fieldset>
                 <label class="level_label">Current Permission Level: </label>
-                <select class="permission-level" name="level">
+                <select class="permission-level" name="level" ng-disabled="user.role === 'network_partner'">
```

Comment:
So long as the canonical source of truth (the API) indicates that a network partner is no longer explicitly supported, that's fine.